packages/advanced-logic/test/extensions/payment-network/any-to-erc20-proxy.test.ts

```diff
@@ -12,7 +12,7 @@ describe('extensions/payment-network/erc20/any-to-erc20-fee-proxy-contract', ()
     it('can create a create action with all parameters', () => {
       // 'extension data is wrong'
       expect(
-        anyToErc20Proxy.createCreationAction({
+        new anyToErc20ProxyPaymentNetwork().createCreationAction({
```

Comment:
why not use the exported default `conversionErc20FeeProxyContract`? I think it would avoid all these changes to the tests